README.md

```diff
@@ -1,5 +1,9 @@
 # Generate Navigational Components With Ease !!!
-[![NPM version][npm-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Dependency Status][daviddm-image]][daviddm-url] [![Coverage Status][coveralls-image]][coveralls-url]
```

Comment:
The rendering of the badges is always on the same line I have added a new line for every badge only to read better the file.

Reply:
this is fine 👍